app/src/main/res/drawable/account_icon.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I'm curious, where did you get these? Looked good, hopefully from the android icons or copyright free source?

Reply:
I got it from https://fonts.google.com/icons